pkg/queue/cluster_queue_test.go

```diff
@@ -86,6 +90,36 @@ func Test_PushOrUpdate(t *testing.T) {
 						Reason: kueue.WorkloadEvictedByPodsReadyTimeout,
 						Status: metav1.ConditionTrue,
 					}).
+					Condition(metav1.Condition{
+						Type:   kueue.WorkloadRequeued,
+						Status: metav1.ConditionFalse,
+					}).
+					Obj()),
+			},
+		},
+		"workload is still have Requeued=true": {
```

Comment:
Is this description accurate, the workload has `Requeued: false`. Please try to capture the intention of the new test case in the name.

Reply:
Thanks. Fixed.